### PR TITLE
Issue #210117 fix: Getting Error while calling updateLearnerProfile api

### PR DESCRIPTION
**Pull request by ajinkyapandetekdi**

### CODE REVIEW

File: src/pages/Story/Story.css [hunk 1]
```diff
@@ -196,7 +196,7 @@
   background: gray.50;	
   background-image: url('../../assests/Images/hackthon-images/login-bg.jpg');	
   background-color: #cccccc;	
-  height: 500px;	
+  /* height: 500px;	 */
```
Comment (gouravmore) [inline src/pages/Story/Story.css:199]: Remove these changes from this PR

**Comment by ajinkyapandetekdi** [comment]
[Getting 500 Error While Calling Api.webm](https://github.com/Sunbird-ALL/all-ll-app/assets/137033511/a56466bb-e062-450c-9086-fdfa9ed3088a)

**Comment by ajinkyapandetekdi** [comment]
previously we were getting 500 errors from API. It is because the Audio file is missing from the Payload. 
After adding the above condition Api worked properly and also passed the Audio file in the payload.

**Comment by gouravmore** [comment]
Merged as a part of the other PR

**Pull request closed without merging**